simulator/simple_dds/__init__.py

```diff
@@ -0,0 +1,2 @@
+__all__ = ["entity", "dds_service", "domain_participant", "publisher", "subscriber",
+"data_object", "topic"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Undefined variable name 'data_object' in __all__](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=5588062)

Reply:
Variable is needed for importing submodules contained in the folder.